recipes/beyla-golden-signals/beyla-daemonset.yaml

```diff
@@ -75,7 +75,24 @@ spec:
           volumeMounts:
           - name: bpffs
             mountPath: /sys/fs/bpf
+          - name: beyla-config
+            mountPath: /config
       volumes:
       - name: bpffs
         hostPath:
           path: /sys/fs/bpf
+      - name: beyla-config
+        configMap:
+          name: beyla-config
+
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: beyla-config
+data:
+  beyla-config.yml: |
+    discovery:
```

Comment:
This could only be configured via YAML. Should we move the other envvars into the YAML?

Reply:
yeah, go for it.

Reply:
I'll do it in a separate PR